bitnami/spring-cloud-dataflow/values.yaml

```diff
@@ -1092,6 +1092,14 @@ mariadb:
       CREATE DATABASE IF NOT EXISTS `skipper`;
       GRANT ALL ON skipper.* to 'skipper'@'%';
       FLUSH PRIVILEGES;
+## Flyway Configuration
+## All database creation scripts are ignored on startup when flyway.enabled is set to false
+## This feature can be used in scenario, where Database tables are already present in Mariadb or ExternalDatabase.
+##
+flyway:
+  ## @param flyway.enabled Enable/disable flyway
+  ##
+  enabled: true
```

Comment:
```suggestion
## Flyway Configuration
## @param flyway.enabled Enable/disable flyway running Dataflow and Skipper Database creation scripts on startup
## All database creation scripts are ignored on startup when flyway.enabled is set to false
## This feature can be used in scenario, where Database tables are already present in Mariadb or ExternalDatabase.
##
flyway:
  enabled: true
```

Please note the table in the README is generated by using this metadata. In my suggestion, I added the sentence you used in the README with the parameter metadata